bip-musig2.mediawiki

```diff
@@ -57,16 +57,13 @@ The MuSig2 variant in this specification stands out by combining all of the foll
 
 == Overview ==
 
-Implementers of the specification must make sure to understand this section thoroughly to avoid subtle mistakes that may lead to catastrophic failure.
+Implementers of this specification must make sure to understand this section thoroughly to avoid subtle mistakes that may lead to catastrophic failure.
 
-This specification is written with a focus on clarity.
-As a result, the specified algorithms are not always optimal in terms of computation and space.
-In particular, some values are recomputed but can be cached in actual implementations (see [[#signing-flow|Signing Flow]]).
+=== Optionality of Features ===
 
-This BIP aims to provide a standard that supports a wide range of possible application scenarios.
-Given a specific application scenario, some features supported by this standard may be unnecessary or not desirable, and implementations can choose not to support them.
-
-Such features include:
+The goal of this specification is support a wide range of possible application scenarios.
+Given a specific application scenario, some features supported by this specification may be unnecessary or not desirable, and implementations can choose not to support them.
```

Comment:
nit: implementations do not have the ability to choose, implementers do

Reply:
ok